v/vert.x/eclipse_vert_x_build_ppc64le_ubi8.4.sh

```diff
@@ -0,0 +1,125 @@
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+PACKAGE_NAME=vert.x
+PACKAGE_PATH=github.com/eclipse-vertx/vert.x
+PACKAGE_VERSION=${1:-3.9.7}
+PACKAGE_URL=https://github.com/eclipse-vertx/vert.x
+
+BUILD_HOME=`pwd`
+
+echo "`date +'%d-%m-%Y %T'` - Staring eclipse vert.x build. Dependencies will be cloned in $BUILD_HOME"
+
+# ------- Install dependencies -------
+
+yum -y install maven make python3
+
+yum -y install subscription-manager.ppc64le
+yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+subscription-manager repos --enable rhel-7-for-power-le-extras-rpms
+yum -y install epel-release
+
+yum -y install gcc-c++.ppc64le
+yum -y install wget
+yum -y install git
+yum -y install java-1.8.0-openjdk-devel
+dnf install openssl-devel.ppc64le -y
+dnf group install "Development Tools" -y
+
+dnf install ninja-build.ppc64le -y
+dnf install golang -y
+dnf install autoconf -y
+dnf install automake -y
+dnf install libtool -y
+dnf install make tar glibc-devel libaio-devel openssl-devel lksctp-tools -y
+dnf install apr-devel apr-util-devel -y
+
+ln -s /usr/bin/python3 /usr/bin/python
+
+echo "`date +'%d-%m-%Y %T'` - Installed Standard Packages -----------------------------------"
+echo "---------------------------------------------------------------------------------------"
+
+# ------- Clone and build source -------
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.312.b07-2.el8_5.ppc64le/
+
+cd $BUILD_HOME
+
+# --------- Installing cmake version 3.21.2 -----------------
+echo "--------- Installing cmake version 3.21.2 -----------------"
+
+wget https://github.com/Kitware/CMake/releases/download/v3.21.2/cmake-3.21.2.tar.gz
+tar -xvf cmake-3.21.2.tar.gz
+cd cmake-3.21.2
+./bootstrap
+make
+make install
+cd ..
+
+
+# --------- Installing ninja version v1.4.0 -----------------
+echo "--------- Installing ninja version v1.4.0 -----------------"
+git clone git://github.com/ninja-build/ninja.git
+cd ninja
+git checkout v1.4.0
+./bootstrap.py
+cp -r ./ninja /usr/bin/
+ninja --version
+cd ..
+
+
+#-------- Building tcnative version 2.0.36.Final ----------------
+echo "`date +'%d-%m-%Y %T'` - Installed Build Dependencies -----------------------------------"
+echo "-------- Building tcnative version 2.0.36.Final ----------------"
+
+git clone --recurse https://github.com/netty/netty-tcnative.git
+cd netty-tcnative
+git checkout netty-tcnative-parent-2.0.36.Final
+
+./mvnw clean install -DskipTests
+cd ..
+
+#-------- Building netty version 4.1.60.Final ----------------
+echo "`date +'%d-%m-%Y %T'` - Installed Build Dependencies -----------------------------------"
+echo "-------- Building netty version 4.1.60.Final ----------------"
+
+git clone --recurse https://github.com/netty/netty
+cd netty
+git checkout netty-4.1.60.Final  # version checkout
+mvn clean install -DskipTests
```

Comment:
why are we skipping the test execution?

Reply:
Skipping these tests as tests are failing and we only need its jar file. Which we get with build only.
There is no impact on targeted module. (i.e., vert.x)